monai/utils/enums.py

```diff
@@ -311,3 +311,15 @@ class JITMetadataKeys(Enum):
     TIMESTAMP = "timestamp"
     VERSION = "version"
     DESCRIPTION = "description"
+
+
+class BoundingBoxMode(Enum):
```

Comment:
I feel this `BoundingBoxMode` is easy to be confused with `BoxMode` class.
Can we change it to `BoxModeName` or some other better class name?

Thanks.